flink-connector-influxdb2/pom.xml

```diff
@@ -30,8 +30,8 @@ under the License.
     <relativePath>..</relativePath>
   </parent>
 
-  <artifactId>flink-next-gen-connector-influxdb_2.12</artifactId>
-  <name>flink-next-gen-connector-influxdb</name>
+  <artifactId>flink-connector-influxdb2_2.12</artifactId>
```

Comment:
Dependencies in der pom.xml auf die neueste Version ziehen (InfluxDB, Flink at least)